app.js

```diff
@@ -59,6 +61,12 @@ io.on('connection', (socket) => {
 		}
 	});
 
+
+	// Welcome the user to the app
+	const welcomeUser = (socket, data) => {
+		io.to(socket.id).emit('welcome user', data)
+	}
+
```

Comment:
Do not make a separate socket for welcome-bot. Just add a user named 'WelcomeBot' and use this user to welcome participants on joining. I want to keep it similar to bots on a discord server.

Reply:
It's not a separate socket, it's the main socket instance passed into the function.

Reply:
I can see that, but I wanted to say that you should just make a permanent user of the app named `WelcomeBot`, and treat it as another person using the app.

Reply:
So you will have many users in your app that potentially say 1 thing and never again are used? I think we should talk about a more efficient way to approach this. One bot to communicate with a user makes more sense.